worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -265,6 +264,13 @@ private[celeborn] class Worker(
     ThreadUtils.newDaemonSingleThreadScheduledExecutor("async-reply")
   val timer = new HashedWheelTimer()
 
+  metricsSystem.registerSource(new ThreadPoolSource(
+      "worker-replicate-data", replicateThreadPool, conf, MetricsSystem.ROLE_WORKER))
+  metricsSystem.registerSource(new ThreadPoolSource(
+    "worker-commit-files", commitThreadPool, conf, MetricsSystem.ROLE_WORKER))
+  metricsSystem.registerSource(new ThreadPoolSource(
+    "worker-clean-expired-shuffle-keys", cleanThreadPool, conf, MetricsSystem.ROLE_WORKER))
```

Comment:
There are a few other thread pools which might benefit from this - like `nettyRpcEnv.clientConnectionExecutor` for example.
Thoughts on adding for them as well ?

Reply:
Yea, will check all usage of thread pool